.bazelrc

```diff
@@ -5,6 +5,10 @@
 # Required until this is the default; expected in Bazel 7
 common --enable_bzlmod
 
+# Required for range requests
```

Comment:
could you give a link or more description? I don't know what a "range request" is.